rest/utilities_testing.go

```diff
@@ -280,7 +280,11 @@ func (rt *RestTester) Bucket() base.Bucket {
 			// If scopes is already set, assume the caller has a plan
 			if rt.DatabaseConfig.Scopes == nil {
 				// Configure non default collections by default
-				rt.DatabaseConfig.Scopes = GetCollectionsConfig(rt.TB, testBucket, rt.numCollections)
+				syncFn := base.StringPtr(rt.SyncFn)
```

Comment:
I feel like this would be more readable as:
```
var syncFn *string
if rt.SyncFn != nil {
   syncFn = base.StringPtr(rt.SyncFn)
}
```
As written it makes me wonder "why are we calling StringPtr when we know we don't care about empty values?"